msg/NewNodeEvent.msg

```diff
@@ -0,0 +1,3 @@
+int32 new_node_id
```

Comment:
Does this also need to include the edge that was created and its covariance? Nodes are not added alone